docs/resources/google_compute_zone.md.erb

```diff
@@ -0,0 +1,56 @@
+---
+title: About the google_compute_zone Resource
+platform: gcp
+---
+
+# google\_compute\_zone
+
+Use the `google_compute_zone` InSpec audit resource to test properties of a single GCP compute zone.
+
+<br>
+
+## Syntax
+
+A `google_compute_zone` resource block declares the tests for a single GCP zone by project and name.
+
+    describe google_compute_zone(project: 'chef-inspec-gcp',  zone: 'us-east1-b') do
+      it { should exist }
+      its('name') { should match 'us-east1-b' }
+    end
+
+<br>
+
+## Examples
+
+The following examples show how to use this InSpec audit resource.
+
+### Test that a GCP compute zone does not exist
+
+    describe google_compute_zone(project: 'chef-inspec-gcp',  zone: 'us-east1-b') do
+      it { should_not exist }
+    end
+
+### Test that a GCP compute zone is in the expected state
+
+    describe google_compute_zone(project: 'chef-inspec-gcp',  zone: 'us-east1-b') do
```

Comment:
This seems like a great chance for a bit of UX sugar.  If you defined a method `def up? ...`, you'd have a `be_up` matcher, and you can say 
```
describe google_compute_zone(...) do
  it { should be_up }
end
```

Reply:
will add this - much more readable